pkg/platform/aws/aws.go

```diff
@@ -31,23 +31,24 @@ import (
 )
 
 type workerPool struct {
-	Name         string            `hcl:"pool_name,label"`
-	Count        int               `hcl:"count"`
-	SSHPubKeys   []string          `hcl:"ssh_pubkeys"`
-	InstanceType string            `hcl:"instance_type,optional"`
-	OSChannel    string            `hcl:"os_channel,optional"`
-	OSVersion    string            `hcl:"os_version,optional"`
-	Labels       map[string]string `hcl:"labels,optional"`
-	Taints       map[string]string `hcl:"taints,optional"`
-	DiskSize     int               `hcl:"disk_size,optional"`
-	DiskType     string            `hcl:"disk_type,optional"`
-	DiskIOPS     int               `hcl:"disk_iops,optional"`
-	SpotPrice    string            `hcl:"spot_price,optional"`
-	TargetGroups []string          `hcl:"target_groups,optional"`
-	CLCSnippets  []string          `hcl:"clc_snippets,optional"`
-	Tags         map[string]string `hcl:"tags,optional"`
-	LBHTTPPort   int               `hcl:"lb_http_port,optional"`
-	LBHTTPSPort  int               `hcl:"lb_https_port,optional"`
+	Name             string            `hcl:"pool_name,label"`
+	Count            int               `hcl:"count"`
+	CPUManagerPolicy string            `hcl:"cpu_manager_policy,optional"`
```

Comment:
Value of this field should be validated I think.

Reply:
done

Reply:
Thanks!